Mosaicstg/climate-united
# Rich Text field entries and assets

# Description

Here are the changes involved in this PR.

- Allow for rendering the following for the Page, Case Study, Event and Post content types:
   - Inline Entry hyperlink: text linked to a entry
   - Inline Asset hyperlink: text linked to image, pdf or file
   - Block Asset entry: Image/file directly embedded
- Update Node packages
   -   and resolve [`framer-motion`](https://www.framer.com/motion/) package issues
- Improve focus management for modal on homepage

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

Here are the instructions to test the changes made here:

1. Update `.env` to connect to the `staging` environment on Contentful
2. Run `npm install`
3. Run `npm run dev`
4. View the following pages:
   - `/work-with-us` to see inline entry and asset links and embedded assets
   - Any Case Study (`/case-study*`) page to verify everything renders as it should
   - Any Event page (`/events/*`) to verify everything renders as it should
   - Any Post page (`/news/*`) to reverify everything renders as it should
5. On the homepage, click any region and close the modal and test that your "focus" is on the recently selected region
   - Note that when you click the overlay to close the modal your "focus" isn't remembered since your click propagates to whatever is under the overlay 

# Checklist:
